rfcs/taskforce/0001-buildkiteci.md

```diff
@@ -0,0 +1,223 @@
+# BuildKite CI Switch
+
+===
+
+# Goal
+
+## Relevant Problems
+
+- CircleCI is expensive because it's hosted (P1)
+- Slow builds (P2)
+  - Our configuration is not scheduled optimally — this leads to slower builds than are actually possible (P2.1)
+    - Parallelism is not granular but rather just at the outskirts (P2.1.1)
+    - Test execution doesn't share artifacts before fanning out (P2.1.2)
+  - Instances are not as large as we'd want them to be (even at the "largest" size) (P2.2)
+  - Test execution is not granular (P2.3)
+- Requires a manual template expansion step (test.py) (P3)
+- Iterating on builds requires running things on our existing build infrastructure (P4)
+- Flaky tests don't have a good story ;; It isn't clear which tests are truly "flaky and we want to ignore the flake" and this failure is important to look at (P5)
+- Nix builds don't work in CircleCI's infrastructure anymore (P6)
```

Comment:
Not exactly important but did you consider using a numbered list for this rather than bulleted with numbers at the end 😛

Reply:
Can't nest numbered lists easily in markdown 😛